src/components/Icon/Icon.js

```diff
@@ -0,0 +1,188 @@
+                                  props.type === "check_off" ? "-402.5px -169.5px" :
+                                  props.type === "check_on" ? "-485px -169px" :
+                                  props.type === "sound_white" ? "-75.5px -276px" :
+                                  props.type === "sound_black" ? "-158.5px -276px" :
+                                  props.type === "all_off" ? "-227px -265px" :
+                                  props.type === "sun_off" ? "-310px -265px" :
+                                  props.type === "mon_off" ? "-391px -265px" :
+                                  props.type === "tue_off" ? "-474px -265px" :
+                                  props.type === "wed_off" ? "-557px -265px" :
+                                  props.type === "thu_off" ? "-638px -265px" :
+                                  props.type === "fri_off" ? "-720px -265px" :
+                                  props.type === "sat_off" ? "-803px -265px" :
+                                  props.type === "exit_white" ? "-75.5px -380px" :
+                                  props.type === "exit_black" ? "-158.5px -380px" :
+                                  props.type === "all_on" ? "-227px -370px" :
+                                  props.type === "sun_on" ? "-310px -370px" :
+                                  props.type === "mon_on" ? "-391px -370px" :
+                                  props.type === "tue_on" ? "-474px -370px" :
+                                  props.type === "wed_on" ? "-557px -370px" :
+                                  props.type === "thu_on" ? "-638px -370px" :
+                                  props.type === "fri_on" ? "-720px -370px" :
+                                  props.type === "sat_on" ? "-803px -370px" :
+                                  props.type === "time_white" ? "-76px -488px" :
+                                  props.type === "time_black" ? "-158px -488px" :
+                                  props.type === "previous" ? "-227px -475px" :
+                                  props.type === "next" ? "-309px -475px" :
+                                  props.type === "minus" ? "-391px -475px" :
+                                  props.type === "plus" ? "-473px -475px" :
+                                  props.type === "add_white" ? "-73px -594px" :
+                                  props.type === "add_black" ? "-158px -594px" :
+                                  props.type === "check_circle" ? "-228px -582px" :
+                                  props.type === "play" ? "-309px -580px" :
+                                  props.type === "cancel_circle" ? "-391px -580px" :
+                                  props.type === "check_border" ? "-474px -580px" :
+                                  props.type === "check_square" ? "-550px -576px" : "0 0"};
+
+`;
+const cx = classNames.bind();
+
+export default function Icon(props) {
+  const { type, alt, style} = props;
```

Comment:
```suggestion
export default function Icon({ type, alt, style, src }) {
```
이렇게 변경하면 더 간결해지지 않을까요?

그리고 아래 jsx 렌더링 하는 부분에서 props에 src도 존재한는 것 같아 추가하였습니다.

Reply:
props중에 src가 들어가는 경우가 있고 안들어가는 경우가 있어서 따로 선언한 것 인데 사용 안할 시에도 문제 없는지 확인해보고 가능하다면 수정하겠습니다.